src/FormBuilder/resources/js/form-builder/src/blocks/fields/text/Edit.tsx

```diff
@@ -3,13 +3,12 @@ import {BlockEditProps} from '@wordpress/blocks';
 
 export default function Edit({attributes}: BlockEditProps<any>) {
     const {label, isRequired, description, placeholder, defaultValue} = attributes;
-    const requiredClass = isRequired ? 'give-is-required' : '';
 
     return (
         <>
```

Comment:
Works for me. 
I noticed we have it wrapped in fragments. Do we need them?

Reply:
Nope! resolved b2ab1131